content/3.workflow/3.participant-onboarding.md

```diff
@@ -15,24 +15,29 @@ To onboard the Participant, the following steps are performed:
 
 ```mermaid
 sequenceDiagram
+  autonumber
+ %% using slide 12
 
-  participant CA as Client Application
-  participant MW as Middleware 
+  participant p as Participant
+  participant ca as Client Application
 
   box Corda Network
-  participant PN as Participant Node
-  participant GN as Gevamu Payments Gateway Node
+  participant pc as Payment CorDapp powered by Gevamu Payment SDK 
+  participant gn as Gevamu Payments Gateway Node
   end
 
-  participant PSP as Bank/PSP
+  participant psp as Bank/PSP
 
-  CA -->> MW: Rest API
+  ca ->> pc: Request to join as a Participant
 
-  MW -->> PN: rpc call to intiate registration flow
+  rect rgb(252,212,213)
+  Note right of pc: On-chain
+  pc ->> gn: send request to the gateway
+  gn ->> pc: Provide Network ID <br/> Assign Participant ID
+  end
 
-  PN ->> GN: registration flow
-  GN ->> PN: returns credentials after successful registration of node 
+  pc ->> ca: Display Network Id and <br/> Participant Id 
```

Comment:
Application can only send information to another application. Information is displayed inside one specific application
```suggestion
  pc ->> ca: Participant's credentials
  pc ->> pc: Display Network Id and <br/> Participant Id 
```

Reply:
Fixed